server/controllers/profile.js

```diff
@@ -0,0 +1,115 @@
+const Profile = require("../models/Profile");
+const asyncHandler = require("express-async-handler");
+
+// @route POST /profiles
+// @desc Given parameters passed in create a profile
+// @access Private
+
+exports.Profiles = asyncHandler(async (req, res, next) => {
+    const { firstname, lastname, description, startdate, enddate } = req.body;
+    const profile = await Profile.create({
+        firstname,
+        lastname,
+        description,
+        startdate,
+        enddate
+      });
+    
+      if (profile) {
+        const token = generateToken(profile._id);
```

Comment:
Please explain why are we generating a token here? (when a user logs in, a token is already being generated)